docs/index.md

```diff
@@ -271,34 +295,76 @@ Follow these steps:
 	```
 
 	Also available is the `vault delete astra/role org_id="<ORG ID>" role="<ROLE NAME>"` command.
+	
+5. Define a default lease time. Example:
+
+	```bash
+	vault write astra/config \
+	org_id="astra_token=AstraCS:YlABLSDOEMpQlrdWoLLJyzAh:8e34d55b6d774a7822ad87df2e502774749dc5549fd7e2bd248af307bee4ca8b" \
+	url=:https://api.astra.datastax.com" logical_name="internalOrg" renewal_time="4h"
```

Comment:
url="https://api.astra.datastax.com" instead of url=:https://api.astra.datastax.com"

Reply:
Good catch - thanks Bansri.